specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/applicationGateway.json

```diff
@@ -2076,6 +2430,148 @@
       ],
       "description": "Redirect configuration of an application gateway."
     },
+    "ApplicationGatewayPrivateLinkResourcePropertiesFormat": {
```

Comment:
>Format [](start = 52, length = 6)

why include the "Format" suffix?  It will make the class/object names in generated SDKs awkward

Reply:
Just followed the naming we have in ApplicationGateway for other child resource properties. I dont mind changing it but for consistency within ApplicationGateway SDKs, is it okay to keep it ?